.github/workflows/autoformat.yml

```diff
@@ -58,14 +59,15 @@ jobs:
     if: ${{ needs.check-comment.outputs.should_autoformat == 'true' }}
     permissions:
       pull-requests: read # view files modified in PR
+    outputs:
+      reformatted: ${{ steps.patch.outputs.reformatted }}
     steps:
       - uses: actions/checkout@v3
         with:
           repository: ${{ needs.check-comment.outputs.repository }}
           ref: ${{ needs.check-comment.outputs.head_ref }}
           # Set fetch-depth to merge the base branch
           fetch-depth: 300
-          token: ${{ secrets.MLFLOW_AUTOMATION_TOKEN }}
```

Comment:
Guess what happens if I file a PR with the following change:

```
diff --git a/requirements/lint-requirements.txt b/requirements/lint-requirements.txt
index fe8bf70f5..1439d0afb 100644
--- a/requirements/lint-requirements.txt
+++ b/requirements/lint-requirements.txt
@@ -1,6 +1,6 @@
 pylint==2.14.4
 ruff==0.0.286
 rstcheck==6.1.1
-black[jupyter]==23.7.0
+git+https://github.com/harupy/black.git
 blacken-docs==1.16.0
 pre-commit==2.20.0
```

and you run `@mlflow-automation autoformat`.